srcpkgs/gimp-lqr-plugin/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'gimp-lqr-plugin'
+pkgname=gimp-lqr-plugin
+version=0.7.2
+revision=1
+wrksrc="${pkgname}-${version}"
```

Comment:
This is already the default, please remove that line.

Reply:
Will do.